lib/challenge.js

```diff
@@ -30,10 +40,14 @@
   // - a name
   // - an address
 
-var pet_owner;
 
-// Answer Starts Here
 
+// Answer Starts Here
+var pet_owner {
+  Owner: "John Doe",
+  name: "Max",
+  address:"21 franklin st."
+}
```

Comment:
`var variableName` declares a new variable. It is itself a complete statement. The variable begins with a value of `undefined`.

```js
{
    prop1: val1,
    prop2: val2
}
```

The above is an object literal. It is a complete value expression.

Following the variable declaration with a value expression, however, is grammatically incorrect. It's like following a noun with another noun in English. There has to be something else in between that describes how they relate, or what to do with one or both of them.

What is missing here (and elsewhere) is the **assignment operator (`=`)**. You want to assign the object literal to the variable `pet_owner`. You achieve this by putting the assignment operator between the variable name and the value expression, like so:

```js
var pet_owner = {
  Owner: "John Doe",
  name: "Max",
  address:"21 franklin st."
};
```

3/4